code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm

```diff
@@ -50,14 +59,53 @@ GLOBAL_LIST_INIT(electrolyzer_reactions, electrolyzer_reactions_list())
 /datum/electrolyzer_reaction/nob_conversion
 	name = "Hypernob conversion"
 	id = "nob_conversion"
+	desc = "Conversion of Hypernoblium into Antinoblium"
 	requirements = list(
 		/datum/gas/hypernoblium = MINIMUM_MOLE_COUNT,
 		"MAX_TEMP" = 150
 	)
+	factor = list(
+		/datum/gas/hypernoblium = "Hypernoblium is consumed at 1 reaction rate",
+		/datum/gas/antinoblium = "Antinoblium is produced at 0.5 reaction rate",
+		"Temperature" = "Can only occur under 150 kelvin.",
+		"Location" = "Can only happen on turfs with an active Electrolyzer.",
+	)
 
 /datum/electrolyzer_reaction/nob_conversion/react(turf/location, datum/gas_mixture/air_mixture, working_power)
 
 	air_mixture.assert_gases(/datum/gas/hypernoblium, /datum/gas/antinoblium)
 	var/proportion = min(air_mixture.gases[/datum/gas/hypernoblium][MOLES], (1.5 * (working_power ** 2)))
 	air_mixture.gases[/datum/gas/hypernoblium][MOLES] -= proportion
 	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.5
+
+/datum/electrolyzer_reaction/halon_generation
+	name = "Halon generation"
+	id = "halon_generation"
+	desc = "Production of halon from CO2 and N2O"
+	requirements = list(
+		/datum/gas/carbon_dioxide = MINIMUM_MOLE_COUNT,
+		/datum/gas/nitrous_oxide = MINIMUM_MOLE_COUNT,
+		"MAX_TEMP" = 230
+	)
+	factor = list(
+		/datum/gas/carbon_dioxide = "CO2 is consumed at 2 reaction rate",
+		/datum/gas/nitrous_oxide = "N2O is consumed at 1 reaction rate",
+		/datum/gas/halon = "Halon is produced at 1 reaction rate",
+		"Energy" = "300 joules of energy is released per reaction rate",
```

Comment:
What is reaction rate? we should talk in a unit that an entry level guy will be comfortable with

Reply:
after some thoughts i don't know if we can have it named in different ways, because is not a "2 moles per tick" situation, the rate depends on other factors and it could even be 21 moles (for example), so 42 moles for CO2 

what should i do?

Reply:
Rather then talking in terms of per tick, why not speak purely in terms of the relationship between consumed gas and output gas?

so for h2o for instance, each mol of h2o creates 2 moles of o2, and 1 mol of hydrogen.
(btw there's something weird goind on with that reaction, it doesn't properly use INVERSE and it also seems to fuck up the h2o converstion ratio)

Reply:
addressed (also is 2 H2O for 1 O2 and 2 H2)